### PR TITLE
This is my pull request for feature 1

### PR DESCRIPTION
**Pull request by algdsapractice**

### CODE REVIEW

File: src/test/java/com/upgrad/BankingSystemTest.java [hunk 1]
```diff
@@ -55,13 +55,17 @@ public void testWithDrawSuccess(){
...
     public void testWithDrawInSufficientFunds(){
         double withDrawAmount = 10000;
-        when(accountMock.getBalance()).thenReturn(6000.0);
 
         double updatedAmount=bankingSystem.withdraw(accountNumber,withDrawAmount);
     }
 
 
 
+    public void testWithDrawInOutOFFunds(){
```
Comment (mithileshse) [inline src/test/java/com/upgrad/BankingSystemTest.java:65]: you have to add the @Test annotation which is missing

File: src/test/java/com/upgrad/TestGit.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package com.upgrad;
+// testing for git
```
Comment (mithileshse) [inline src/test/java/com/upgrad/TestGit.java]: This is class is not required its an empty class please remove it

**Review by mithileshse — CHANGES_REQUESTED** [review]

File: src/test/java/com/upgrad/TestGit.java (hunk 2 above)
Comment (mithileshse) [reply src/test/java/com/upgrad/TestGit.java]: Deleted this class

File: src/test/java/com/upgrad/BankingSystemTest.java (hunk 1 above)
Comment (algdsapractice) [reply src/test/java/com/upgrad/BankingSystemTest.java:65]: Test annotation is added

**Pull request merged**